python/nav/smsd/uninettmaildispatcher.py

```diff
@@ -25,8 +25,7 @@
 
 from django.core.mail import EmailMessage
 
-# pylint: disable=unused-import
-from nav import models as _  # ensure Django is setup for NAV
+from nav import models as _  # noqa: F401 - ensure Django is setup for NAV
```

Comment:
I suspect this might also be obsolete today.  We should check...

Reply:
I checked, and it looks obsolete.  The sms daemon was not originally built with Django, so our "modern" Django bootstrap was not included in the process setup until 2019.  Now Django is bootstrapped when `smsd` starts, so this works just fine without this silly import.